### DISCUSSION TITLE
微信和腾讯会议安装的指引需要更新

### DISCUSSION THREAD

**Discussion by zhangdzhang** [NakanoMikuOrg/arch-guide#84] [General]
腾讯会议：
[zhangdan@danendeavourosx240 ~]$ yay -S com.tencent.meeting.deepin
 -> 找不到所有需要的包:
	com.tencent.meeting.deepin (目标)

微信：
[zhangdan@danendeavourosx240 ~]$ yay -S com.qq.weixin.deepin
 -> 找不到所有需要的包:
	deepin-udis86 (被这些包需要: com.qq.weixin.deepin -> deepin-wine6-stable)

或者我需要调整什么？我的系统是endeavouros,除了主题和自带工具之类，其他和arch应该是一模一样的
[zhangdan@danendeavourosx240 ~]$ neofetch
        #####           zhangdan@danendeavourosx240 
       #######          --------------------------- 
       ##O#O##          OS: x86_64 
       #######          Host: 20AMA3A3CD ThinkPad X240 
     ###########        Kernel: 5.15.77-1-lts 
    #############       Uptime: 35 mins 
   ###############      Packages: 1109 (pacman) 
   ################     Shell: bash 5.1.16 
  #################     Resolution: 1366x768 
#################     WM Theme: Arc-Darker 
                        Theme: Arc-Dark [GTK2], Arc-Darker [GTK3] 
                        Icons: Papirus-Dark [GTK2], Qogir [GTK3] 
                        Terminal: xfce4-terminal 
                        Terminal Font: Source Code Pro 10 
                        CPU: Intel i5-4200U (4) @ 2.600GHz 
                        GPU: Intel Haswell-ULT 
                        Memory: 2278MiB / 7641MiB 

谢谢～

**Comment by Errant404** [comment]
微信安装所需依赖请参见https://github.com/NakanoMikuOrg/arch-guide/issues/86#issuecomment-1354321013
腾讯会议包名已修正为`wemeet-bin` ，请使用`yay -S wemeet-bin` 安装

**Comment by zhangdzhang** [comment]
谢谢！

在 12/20/22 02:36, Errant 写道:
